pkg/features/kube_features.go

```diff
@@ -727,8 +728,8 @@ var defaultKubernetesFeatureGates = map[featuregate.Feature]featuregate.FeatureS
 	CSIMigrationvSphere:            {Default: false, PreRelease: featuregate.Beta},
 	CSIMigrationvSphereComplete:    {Default: false, PreRelease: featuregate.Beta},
 	RunAsGroup:                     {Default: true, PreRelease: featuregate.Beta},
-	CSIMigrationOpenStack:          {Default: false, PreRelease: featuregate.Beta}, // Off by default (requires OpenStack Cinder CSI driver)
-	CSIMigrationOpenStackComplete:  {Default: false, PreRelease: featuregate.Alpha},
+	CSIMigrationOpenStack:          {Default: true, PreRelease: featuregate.Beta},
+	CSIMigrationOpenStackComplete:  {Default: true, PreRelease: featuregate.Beta},
```

Comment:
@dims Please rebase.

And please don't set `InTreePluginOpenStackUnregister` (former `CSIMigrationOpenStackComplete
`)  to true - it's alpha and IIRC can't be enabled until CSI migration in GA + 2 releases, because Attach/Detach controller in kube-controller-manager needs to support attaching volumes to 2 release old kubelets, where CSI migration was not GA and they may use in-tree volume plugin. That requires  in-tree cloud provider + volume plugin to be in KCM for some releases.

Reply:
gotcha!